substrate/bin/node/runtime/src/lib.rs

```diff
@@ -2196,7 +2196,7 @@ pub type Executive = frame_executive::Executive<
 // All migrations executed on runtime upgrade as a nested tuple of types implementing
 // `OnRuntimeUpgrade`.
```

Comment:
Maybe also put a comment that these are just examples and do not need to run directly after the genesis block?

Reply:
Done.